### ISSUE TITLE
Create a published specification version page and some form of banner

### ISSUE THREAD

**Issue by james-answer** [nhsconnect/gpconnect#337]
Create a specification page to give access to older published versions of the specification.

Also add a banner to show if the specification being viewed is the current version of the specification or an older version of the specification with a link to the specification published versions page.

**Comment by james-answer** [comment]
I have added a new page for the release specification versions and added a banner at the top of the page to show the user if they are looking at the current or an old version of the specification. The aim is to use the jenkins build to change the CSS for previous builds in order to show the historical version banner rather than the current banner.

**Comment by james-answer** [comment]
Added a link to the test pack release for each spec release to page.

**Comment by james-answer** [comment]
After getting access to the Jenkins box which builds the developer network version of the GP Connect specification I had to change the method which is used to display the banner. The script which builds the jekyll pages is shared between multiple projects so the change needed to be generic. I have added functionality which allows the jenkins job to specify a file path to a file containing html and include the html on the page at the top of the api content section.

So that the banner appears on the github pages and netlify I have added some javascript which injects the html into the page. The javascript file is removed by the jenkins build so the banner will not be added on the developer network and the injected banner by the jenkins script will only show.

I have added a released versions page which is linked to by the version banner. This outlines the released versions and some information about how the specification is versioned.
